components/charts/composed-chart/component.jsx

```diff
@@ -133,13 +170,22 @@ class CustomComposedChart extends PureComponent {
     if (isVertical) rightMargin = 10;
     if (!simple && rightYAxis) rightMargin = 70;
 
+    const { barGap = '10%' } = xAxis; // default is 10% according to recharts docs
+    const hasLabels = xAxis?.label || yAxis?.label;
+
     return (
       <div
-        className={cx('c-composed-chart', className)}
-        style={{ height: simple ? 110 : height || 250 }}
+        className={cx('c-composed-chart', className, {
+          'overflow-x-visible': hasLabels,
+        })}
+        style={{
+          height: simple ? 110 : height || 250,
+          paddingLeft: config?.yAxis?.label ? '1%' : '0',
```

Comment:
I would raise the `paddingLeft` a little bit, maybe to `3%`. It would be more like the design example on the ticket and would look less cramped.